.pre-commit-config.yaml

```diff
@@ -32,7 +32,7 @@ repos:
       - id: blacken-docs
         additional_dependencies: [black==22.6]
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v2.7.1"
+    rev: "v3.0.0-alpha.0"
```

Comment:
Are we doing an alpha here?

Reply:
Sorry, it was auto-merged before I could read your comment.

It didn't change the formatting of any of our files so I don't suppose it really matters.